www/js/app.js

```diff
@@ -17,4 +17,9 @@ $(document).ready(function() {
 		advSearch.slideToggle(0);
 		return false;
 	});
+
+	$('.invert-selection').on('click', function() {
+		$(this).parent().find('input').click();
+		return false;
+	});
```

Comment:
Souhlas, this.checked bude čistčí než klik, nicméně to cos napsal také není úplně vhodné - invertuje to všechny checkboxy ve formuláři. Myslím, že ideální by bylo obalit checkoxy prvkem s třídou selection-group a pak filtrovat jen checkboxy v tom.

Ad stream, všiml jsem si toho až po pull requestu :) nicméně si myslím, že typický use case je takový, že vyhledáš z homepage, zjistíš že tam máš plno nezajímavých příspěvků, tak se rozhodneš filtrovat (se streamem je to celé další vyhledávání).

Pokud o to bude zájem, tak to upravím a udělám znovu pull request, pokud ne, tak díky za radu, určitě se bude hodit.